merge_test.go

```diff
@@ -265,3 +263,23 @@ func TestMergeFiles__invalid(t *testing.T) {
 		t.Errorf("expected error: len(out)=%d error=%v", len(out), err)
 	}
 }
+
+func populateFileWithMockBatches(noOfBatches int, file *File) {
+	var startSeq = file.Batches[0].GetHeader().BatchNumber + 1
+	var entryDetail = file.Batches[0].GetEntries()[0]
+	for i := startSeq; i < (noOfBatches + startSeq); i++ {
+		header := mockBatchHeader()
+		header.StandardEntryClassCode = "PPD"
+		header.ServiceClassCode = 225
+		header.CompanyName = "Example Company"
+		header.CompanyIdentification = "132465"
+		header.CompanyEntryDescription = "Example Description"
+		header.ODFIIdentification = "12104288"
+		batch, _ := NewBatch(header)
```

Comment:
I'm not really sure what you mean to check here? Could you explain a bit more?

Reply:
since this is a test helper, you can change the method signature to 
```go
func populateFileWithMockBatches(t *testing.T, numBatches int, file *File)
...
batch, err := NewBatch(header)
if err != nil {
  t.Fatal(err)
...
```